node/pkg/watchers/solana/client.go

```diff
@@ -215,21 +222,27 @@ func NewSolanaWatcher(
 	queryReqC <-chan *query.PerChainQueryInternal,
 	queryResponseC chan<- *query.PerChainQueryResponseInternal,
 ) *SolanaWatcher {
+	msgObservedLogLevel := zapcore.DebugLevel
+	if chainID != vaa.ChainIDPythNet {
```

Comment:
this did confuse me the first time I read it. might be easier to read to have the default be info and then if it *is* pythnet, set it to debug.